Gems/Atom/RHI/DX12/Code/Source/RHI/Fence.h

```diff
@@ -7,12 +7,13 @@
  */
 #pragma once
 
+#include <RHI/DX12.h>
+
```

Comment:
Does this need the same comment as CommandListBase.h about moving the DX12 include first?